src/org/apache/xerces/parsers/DTDParser.java

```diff
@@ -29,6 +29,8 @@
 import org.apache.xerces.xni.parser.XMLDTDScanner;
 
 /**
+ * DTD Parser
```

Comment:
I think I generally prefer no comment to one that more-or-less repeats the class/method/field name. This sweeps the warning under the rug but it doesn't fix the real issue.